javascript/javascript1/week4/homework6/scripts.js

```diff
@@ -0,0 +1,133 @@
+let todos = []; // initializing an empty list of todo
+let loggedName = []; //initializing an empty list of names
```

Comment:
Thank you for the valuable feedback.  
My first implementation was with a string, however, I thought it would give a better "experience" if I stored all entered names, instead of only the latest.